src/cli.zig

```diff
@@ -819,16 +868,16 @@ pub const PrintBundleCommand = struct {
         var stdout = std.io.getStdOut();
 
         var input = try std.fs.openFileAbsolute(try std.os.realpath(entry_point, &out_buffer), .{ .mode = .read_only });
-        const params = comptime [_]Arguments.ParamType{
-            clap.parseParam("--summary  Peek inside the .bun") catch unreachable,
-        };
+        const params = comptime clap.parseParamsComptime(
+            \\--summary  Peek inside the .bun"
+        );
 
-        var jsBundleArgs = clap.parse(clap.Help, &params, .{ .allocator = ctx.allocator }) catch {
+        var jsBundleArgs = clap.parse(clap.Help, &params, Arguments.parser, .{ .allocator = ctx.allocator }) catch {
```

Comment:
I think we can deinit `jsBundleArgs` because the clap memory isn't used later

Reply:
Good point, added.